### PR TITLE
monitoring for tls certificate expiry

### PR DESCRIPTION
**Pull request by Atul-source**
Signed-off-by: a0p0ie5 <a0p0ie5@homeoffice.wal-mart.com>

### CODE REVIEW

**Comment by Atul-source** [comment]
This PR is for issue #62

**Review by sanfern — APPROVED** [review]
LGTM

File: config/l3afd.cfg [hunk 1]
```diff
@@ -68,3 +68,5 @@ cert-dir: /etc/l3af/certs
 cacert-filename: ca.pem
 server-crt-filename: server.crt
 server-key-filename: server.key
+cert-expiry-warning-days: 30
+# how many days before expiry you want warning
```
Comment (dthaler) [inline config/l3afd.cfg:72]: Thanks for at least adding this comment.
FYI, I filed https://github.com/l3af-project/l3afd/issues/79

**Review by dthaler — APPROVED** [review]

**Review by jniesz — APPROVED** [review]
LGTM

**Pull request merged**